pgmpy/models/MarkovChain.py

```diff
@@ -367,6 +369,48 @@ def generate_sample(self, start_state=None, size=1):
                 self.state[j] = State(var, next_st)
             yield self.state[:]
 
+    def check_stationarity(self, tolerance=0.2, sample=None):
```

Comment:
@kris-singh @ankurankan IMO instead of raising a `ValueError` when markov chain is not stationary, it should return a Boolean value. Or is there any particular reason for this design?